doc/source/whatsnew/v1.2.0.rst

```diff
@@ -681,6 +681,7 @@ Plotting
   indexed by a :class:`TimedeltaIndex` with a fixed frequency when x-axis lower limit was greater than upper limit (:issue:`37454`)
 - Bug in :meth:`DataFrameGroupBy.boxplot` when ``subplots=False``, a KeyError would raise (:issue:`16748`)
 - Bug in :meth:`DataFrame.plot` and :meth:`Series.plot` was overwriting matplotlib's shared y axes behaviour when no sharey parameter was passed (:issue:`37942`)
+- Bug in :meth:`DataFrame.plot` to handle nullable integers (:issue:`32073`)
```

Comment:
We want to be more specific:
```
Bug in :meth:`DataFrame.plot` was raising a ``TypeError`` with ``ExtensionDtype`` columns (:issue:`32073`)
```

Reply:
Thank you @arw2019 for your comments. I will add these changes to the next commit.

Reply:
@arw2019 I added the changes to the latest commit. Can you please review it again ?